config/whitelabel.yml

```diff
@@ -90,6 +90,9 @@
   label_id: cologne
   default_locale: de
   recurring: third wednesday
+  custom_recurring:
+    :de: "jeweils am 3. Mittwoch in jedem 2. Monat (Januar, März, Mai, Juli, September, November) um 19:00 Uhr"
+    :en: "every 3rd Wednesday in every second month (January, March, May, July, September, November)"
```

Comment:
makes a lot of sense to put the text into an i18n locale file.  I did so in 6e199de1960ec1047e192a3965e64c57aec9eae9